src/otx/v2/adapters/torch/mmengine/engine.py

```diff
@@ -649,6 +644,24 @@ def _update_codebase_config(
             codebase(str): mmX codebase framework
             task(str): mmdeploy task
         """
-        codebase = codebase if codebase is not None else self.registry.name
-        codebase_config = {"type": codebase, "task": task}
+        codebase = self.registry.name
+        mmdeploy_task_dict = {
+            TaskType.SEGMENTATION: "Segmentation",
+            TaskType.CLASSIFICATION: "Classification",
+            TaskType.DETECTION: "ObjectDetection",
+            TaskType.INSTANCE_SEGMENTATION: "InstanceSegmentation",
+        }
```

Comment:
This may not be directly related to this comment,
I think the `TaskType` is a necessary type internally, but not a good idea for users to use directly. `TaskType` should only be used internally and the user should be able to send and receive it as an input like str. What do you think?